battle-service/src/main/java/org/astashonok/battleservice/constants/BattleConstants.java

```diff
@@ -0,0 +1,14 @@
+package org.astashonok.battleservice.constants;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class BattleConstants {
+
+    public static final int GAME_BOARD_HEIGHT_DEFAULT = 3;
```

Comment:
is it possible to have not square configuration ? i.e. 3x5 10x1?

Reply:
yes) But the game board height and width values must be more or equals 3. The app has checking via validator

Reply:
do you have logic to calculate diagonales in case of non-standard board?

Reply:
yes. App Logic handles non-standard board. When we create Battle object we passed 'winningNumberInRow' property. This property characterizes the number of consecutive matching crosses or zeroes. And When I go around game board I consider row and column numbers. I used the following principle:
![image](https://user-images.githubusercontent.com/34896952/172195391-894ae956-5169-4a1c-aa24-02ebaee29d81.png)